src/main/java/org/wso2/carbon/module/swiftiso20022/utils/JsonToMt103Utils.java

```diff
@@ -0,0 +1,161 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.module.swiftiso20022.utils;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.json.JSONObject;
+import org.wso2.carbon.module.swiftiso20022.constants.ConnectorConstants;
+import org.wso2.carbon.module.swiftiso20022.constants.MT103Constants;
+import org.wso2.carbon.module.swiftiso20022.validation.JsonToMT103PayloadValidator;
+import org.wso2.carbon.module.swiftiso20022.validation.common.ValidationResult;
+
+
+/**
+ * Class with Json to Mt103 utility methods.
+ */
+public class JsonToMt103Utils {
+
+    private static final Log logger = LogFactory.getLog(JsonToMt103Utils.class);
+
+    public static ValidationResult validateBlock01(JSONObject block01Json) {
```

Comment:
Add method comments for all the methods, Check all the files

Reply:
Added in commit 96d4dc498094daee16277501378ee52fbb484dd1